gui/app/controllers/updatesController.js

```diff
@@ -4,7 +4,20 @@
  
  angular
    .module('firebotApp')
-   .controller('updatesController', function($scope) {
+   .controller('updatesController', function($sce, $scope, updatesService) {
+      $scope.updates = {};
 
+      // Get update information
+      updatesService.updateCheck()
+        .then(update => {
+          $scope.updates = update;
+
+          // This is required by angular to strip out bad HTML that could cause issues.
+          $scope.updates.gitNotes = $sce.trustAsHtml($scope.updates.gitNotes)
+
+          // Need this as apparently angular doesn't handle promises well and you have to
+          // manually force it to update after variables change.
+          $scope.$apply();
```

Comment:
Take a look at the comment I leave for the updatesService to see how you can avoid using $scope.$apply.

The reason regular promises don't cause Angular to update is because Angular uses something called "dirty updating" to check if variables have changed. Look it up if you're interested, but in our case we just need to use Angular's implementation of the Promise, which properly triggers scope updates.